.travis.yml

```diff
@@ -23,6 +23,14 @@ matrix:
   include:
     - node_js: 'lts/*'
       env: PRETEST=true
+    - node_js: '9'
+      env: TEST=true ESLINT=5
+    - node_js: '8'
+      env: TEST=true ESLINT=5
+    - node_js: '7'
+      env: TEST=true ESLINT=5
+    - node_js: '6'
+      env: TEST=true ESLINT=5
```

Comment:
sorry for being unclear :-) now we’re not testing eslint 4. Can you change al the “next”s to “4”s?

Reply:
Sure, fixed in d0bebd7da5a1805e57be54106c9b575e37a637da :)